extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/VaultClient.java

```diff
@@ -36,4 +49,21 @@
 
     VaultDatabaseCredentials generateDatabaseCredentials(String token, String databaseCredentialsRole);
 
+    VaultTransitEncrypt encrypt(String token, String keyName, VaultTransitEncryptBody body);
+
+    VaultTransitDecrypt decrypt(String token, String keyName, VaultTransitDecryptBody body);
+
+    VaultTransitSign sign(String token, String keyName, String hashAlgorithm,
+            VaultTransitSignBody body);
+
+    VaultTransitVerify verify(String token, String keyName, String hashAlgorithm,
+            VaultTransitVerifyBody body);
+
+    VaultTransitEncrypt rewrap(String token, String keyName, VaultTransitRewrapBody body);
+
+    void rotate(String token, String keyName);
```

Comment:
Please keep these 3 methods in the Test VaultClient for now, same as we did it earlier for other Vault methods like init since they are not used in the main source.

Reply:
done